### PR TITLE
Convert project filters to dropdown on smaller screens

### PR DESCRIPTION
**Pull request by jgabitto**
- Closes https://github.com/cityofaustin/atd-data-tech/issues/4804
- I deviated a little bit from the mockups Patrick made because I wanted to move forward with my other tasks so I will revisit this in the future.

### CODE REVIEW

**Comment by johnclary** [comment]
@jgabitto looks fantastic in the deploy preview! i don't have time to review the code so i'm approving for product only. thanks @patrickm02L for your input—feel free too open a backlog if issue if you see additional refinements here.

**Review by johnclary — APPROVED** [review]
✨ ⛵

**Review by patrickm02L — APPROVED** [review]
Like how the Progress filters mirror the Workgroup filters. Nice change.

Ship it!

**Review by sergiogcx — APPROVED** [review]
Classy style! 😎 👍

File: src/views/projectsView/ProjectFilters.jsx [hunk 1]
```diff
@@ -51,6 +53,7 @@ function formatSearchPath(currentFilters) {
 }
 
 export default function ProjectFilters(props) {
+  const isTabletOrMobile = useMediaQuery({ query: "(max-width: 996px)" });
```
Comment (mateoclarke) [inline src/views/projectsView/ProjectFilters.jsx:56]: Adding this comment after the fact, but one thing that would be good to consider for this project is to put values like `996px` that are likely to be reused in the future into a constants file. These types of values are often referred to as "magic numbers" and are considered a software anti-pattern. Magic numbers are hard-coded values that appear in code and lack context. Often, its better to assign such values a name and reuse that named value in multiple places. That way, if a breakpoint changes in the future, its easy to update in one place. [Here's one explanation.](https://jeffreyeverhart.com/2020/09/28/javascript-tips-using-named-constants-for-magic-numbers/) 

One example is what we did here for VZE colors: https://github.com/cityofaustin/atd-vz-data/blob/master/atd-vze/src/styles/colors.js.

**Review by mateoclarke — APPROVED** [review]

File: src/views/projectsView/ProjectFilters.jsx (hunk 1 above)
Comment (jgabitto) [reply src/views/projectsView/ProjectFilters.jsx:56]: @mateoclarke Should I go in and add this value to a constants file?

Comment (johnclary) [reply src/views/projectsView/ProjectFilters.jsx:56]: @jgabitto if you haven't talked about this with @mateoclarke already, go ahead and create an issue in atd-data-tech repo and use the `Type: Tech Debt` label. Remind us about it at next sprint planning and we'll queue it up for you.

**Pull request merged**